src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -750,3 +752,45 @@ def get_local_connections_from_executable(executable):
             # ignore when connection not found since it can be configured with env var.
             raise Exception(f"Connection {n!r} required for flow {executable.name!r} is not found.")
     return result
+
+
+def get_current_env_name():
```

Comment:
why not directly identify by the python.exe path? so no need to understand it's a conda or virual env or something else?

Reply:
Good suggestion

Reply:
Remove temporarily for this pr, will put to next prs